templates/repo/settings/options.tmpl

```diff
@@ -170,7 +170,7 @@
 							{{$address := MirrorRemoteAddress .}}
 							<td>{{$address.Address}}</td>
 							<td>{{$.i18n.Tr "repo.settings.mirror_settings.direction.push"}}</td>
-							<td>{{if .LastUpdateUnix}}{{.LastUpdateUnix.AsTime}}{{else}}{{$.i18n.Tr "never"}}{{end}} {{if .LastError}}<div class="ui red label" data-tooltip="{{.LastError}}">{{$.i18n.Tr "error"}}</div>{{end}}</td>
+							<td>{{if .LastUpdateUnix}}{{.LastUpdateUnix.AsTime}}{{else}}{{$.i18n.Tr "never"}}{{end}} {{if .LastError}}<div class="ui red label poping up" data-content="{{.LastError}}">{{$.i18n.Tr "error"}}</div>{{end}}</td>
```

Comment:
Will the style class `poping` ever be renamed to `popping`?
It simply sounds like a typo…
Or do I misunderstand that style class?

Reply:
Actually I think this `poping up` stuff might be a ideocraty we introduced and I think it's actually a complete noop and can be removed, Fomantic only cares about `data-content`. I'm just following the tradition in adding it 😆

Reply:
Actually, there seems to be some JS functionality linked to `poping`, we can just rename it.